openshift/README.md

```diff
@@ -0,0 +1,64 @@
+OpenShift 3 Jenkins
+=========================
+This sample walks through the process of starting up an OpenShift cluster and deploying a Jenkins Pod in it.
+
+Steps
+-----
+
+1. Unless you have built OpenShift locally, be sure  to grab the [oc command, v1.3+](https://github.com/openshift/origin/releases/tag/v1.3.1)
+
+1. Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+        $ oc cluster up
+
+1. Login as a normal user (any non-empty user name and password is fine)
+
+        $ oc login
+
+1. Create a project  named "test"
+
+        $ oc new-project test
+
+1. Run this command to instantiate a Jenkins server and service account in your project:
+
+    If your have persistent volumes available in your cluster:
+
+        $ oc new-app jenkins-persistent
+
+    Otherwise:
+
+    **Note**: jenkins-persistent-template.json template file requires persistent volume setup. 
+    To setup sample persistent volume on new cluster use:
+    
+        $ oc create -f ./sample-pv.json
```

Comment:
@wtrocki I did it like this
```
oc login -u system:admin
oc create -f openshift/sample-pv.json
```
and this time it worked.

Maybe it worked because I had a BeachBallOfDeath and needed to reboot my Mac, or maybe I was trying it wrong before.
I hate computers